extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItem.java

```diff
@@ -108,7 +110,7 @@ public static PhotoModel convertToPhotoModel(
 
   private static String getMimeType(GoogleMediaItem mediaItem) {
     String guessedMimetype = guessMimeTypeFromFilename(mediaItem.getFilename());
-    if (!Strings.isNullOrEmpty(guessedMimetype)) {
+    if (!(Strings.isNullOrEmpty(guessedMimetype) || guessedMimetype.equals(DEFAULT_NULL_MIMETYPE))) {
```

Comment:
where's this DEFAULT_NULL_MIMETYPE being set? Is this needed?

Reply:
Its the default that Tika uses for when it cannot figure out the mime-type. I can add a comment

Reply:
Do we need to check for that here though? I think the !Strings.isNullOrEmpty() clause will always be true if this clause is true so it wont be checked?

Reply:
OOHH sorry I misread and didnt see the extra parenthesis that wrapped this  

This is a little confusing, can we do the check for the DEFAULT_MIME_TYPE in the `guessMimeTypeFromFilename`  function directly? since tahts closer to the TIKA logic?

Reply:
Good call! Moved it, and renamed "DEFAULT_NULL_MIMETYPE" to "DEFAULT_BINARY_MIMETYPE". 
I think much more readable now